monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -370,16 +370,23 @@ class ReportPageComponent extends AuthComponent {
     if (this.state.report.overview.issues[this.Issue.ZEROLOGON_PASSWORD_RESTORE_FAILED]) {
       zerologonOverview.push(<span>
         <WarningIcon/> Automatic password restoration on a domain controller failed!
-        <Button variant={"link"} href={"#"} target={"_blank"} className={"security-report-link"}>
-        Restore your domain controller's password manually.</Button>
+        <Button variant={"link"}
+                href={"https://www.guardicore.com/infectionmonkey/docs/reference/exploiters/zerologon/"}
+                target={"_blank"}
+                className={"security-report-link"}>
```

Comment:
Do the closing '>' characters go on the same line as shown here, or on the next line as shown on line 952?